src/libmeasurement_kit/ooni/templates.cpp

```diff
@@ -21,13 +21,15 @@ void dns_query(Var<Entry> entry, dns::QueryType query_type,
     std::string resolver_hostname, nameserver_part;
     std::stringstream nameserver_ss(nameserver);
 
+    // TODO: Fix IPv6 `nameserver`, use google's [2001:4860:4860::8888]:53 for testing.
```

Comment:
IMHO, merging without further fixes is okayish way to go.

That's especially true, given the uncertain future of libevent-based DNS parser that will be replaced with c-ares. Also c-ares will have alike issue as well, as we've deduced from the API that takes only `in_addr` and `in6_addr`. :)